registry/README.md

```diff
@@ -4,37 +4,37 @@
 
 ### Build Package
 
-Create `azpkg` and upload to a ßstorage account.
-
-```powershell
-PackageGenerator\AzureGalleryPackager.exe -m <path-to-manifest.json> -o <output-dir>
-```
+Use the `make release` command to create `azpkg` file for Docker Container Registry Marketplace Item, and find it in the "_out" folder. Upload the file to a blob container in a storage account, and make sure to change the access policy of the blob to enable anonymous access.
 
 ### Side-load Package
 
-Connect to management endpoint and install package.
+Update admin credential and storage account information in the `gallery.ps1` script, and use it to connect to management endpoint and install package.
 
 ```powershell
 Scripts\gallery.ps1
 ```
 
 ## Solution Pre-requisites
 
-### Create Self-signed Certificate (optional)
+### Create Self-signed Certificate
+
+Update certificate information in the "self-signed.ps1" script, and use it to create a new certificate in `pfx` formatand and export it. If an existing certificate is preferred, use the `Export-PfxCertificate` command instead to export the certificate.
 
 ```powershell
 Scripts\self-signed.ps1
 ```
 
+This certificate will be uploaded to a key vault in a following step, and used by the private docker registry vm.
```

Comment:
- casing: `Key Vault` instance
- subsequent, not following
- virtual machine instead of vm